detectable/src/main/java/com/synopsys/integration/detectable/detectables/maven/cli/MavenCodeLocationPackager.java

```diff
@@ -202,6 +202,10 @@ private void populateGraphDependencies(ExcludedIncludedWildcardFilter scopeFilte
                 dependencyParentStack.push(dependency);
             }
         }
+        shadedDependenciesStuff(dependency);
+    }
+
+    private void shadedDependenciesStuff(ScopedDependency dependency) {
```

Comment:
Simply moved the last chunk of code that was added in the commit form yesterday @devmehtasynopsys, this does resolve the complaint. If you're okay with this and don't believe this will cause any issues I don't see, please suggest a more appropriate method name (I lack context for these changes).  

Sonar issue: https://sonarcloud.io/project/issues?impactSeverities=HIGH&resolved=false&severities=CRITICAL&id=com.synopsys.integration%3Asynopsys-detect&open=AY5erg-0_RLhWwYvLB2A

Reply:
Yeah this would be okay to do and it won't cause any issues. The appropriate method name for this would addShadedDependenciesToGraph, as we are adding them to the graph.